engine/src/main/scala/cromwell/services/ServiceRegistryActor.scala

```diff
@@ -0,0 +1,51 @@
+package cromwell.services
+
+import akka.actor.{Actor, ActorRef, Props}
+import com.typesafe.config.{ConfigFactory, Config, ConfigObject}
+import cromwell.engine.db.DataAccess
+import cromwell.services.ServiceRegistryActor._
+import lenthall.config.ScalaConfig._
+
+import scala.collection.JavaConverters._
+
+object ServiceRegistryActor {
+  case class ServiceRegistryFailure(serviceName: String)
+  trait ServiceRegistryMessage {
+    def serviceName: String
+  }
+
+  def props(dataAccess: DataAccess, config: Config) = {
+    Props(ServiceRegistryActor(dataAccess, config))
+  }
+}
+
+case class ServiceRegistryActor(dataAccess: DataAccess, globalConfig: Config) extends Actor {
```

Comment:
I don't know about other people here, but personally I prefer this one more than the previous one. Some further improvements can be: 
1.) To use Props's reflection thingy here instead of explicit method invocation (something like [this](https://github.com/broadinstitute/cromwell-backend/blob/c7440eee7ddf34599ff5abff52e6d3e44a8efa0d/src/main/scala/cromwell/backend/DefaultBackendFactory.scala#L15) that I previously found simpler to use) 

2.) Have that `DataAccess.globalDataAccess` reference directly in the KVServiceActor instead of passing it in the props (since ideally in a distant utopia, that actor should be responsible for creating it's own DB access stuff)